### PR TITLE
Added tests for inactive blog checking (#197)

### PR DESCRIPTION
**Pull request by Silvyre**
This pull request adds testing for the `check()` function defined in [inactive-blog-filter.js](https://github.com/Seneca-CDOT/telescope/blob/master/src/inactive-blog-filter.js#L41).

This pull request does *not*, however, cover testing for the `update()` function. As noted in https://github.com/Seneca-CDOT/telescope/issues/197#issuecomment-554145637, the creation of some testing environment, or some alteration of the `update()` function needs to be done to facilitate proper testing. I plan to continue looking into this, and plan to make a separate pull request when I come up with a solution.

### CODE REVIEW

File: test/inactive-blog-filter.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const inactiveFilter = require('../src/inactive-blog-filter');
+
+const redlist = JSON.parse(fs.readFileSync('feeds-redlist.json', 'utf8'));
```
Comment (humphd) [inline test/inactive-blog-filter.test.js]: Change this to `const redlist = require('feeds-redlist.json');`

File: test/inactive-blog-filter.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+describe('Redlisted feed checking', () => {
+  it('should return false for bad feed URLs', async () => {
```
Comment (humphd) [inline test/inactive-blog-filter.test.js]: You can remove `async` here, everything you're doing is sync

File: test/inactive-blog-filter.test.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    const badURLs = [redlist[0].url.slice(0, -1), '', undefined];
+    badURLs.forEach((url) => {
+      inactiveFilter.check(url, (callback, result) => {
+        expect(result).toBe(false);
+      });
+    });
+  });
+
+  it('should return true for all feed URLs in feeds-redlist.json', async () => {
```
Comment (humphd) [inline test/inactive-blog-filter.test.js]: Same thing here, drop `async`

File: test/inactive-blog-filter.test.js (hunk 1 above)
Comment (Silvyre) [reply test/inactive-blog-filter.test.js]: Thank you for the review!
(Resolved by b8019ac.)

File: test/inactive-blog-filter.test.js (hunk 2 above)
Comment (Silvyre) [reply test/inactive-blog-filter.test.js]: (Resolved by b8019ac.)

File: test/inactive-blog-filter.test.js (hunk 3 above)
Comment (Silvyre) [reply test/inactive-blog-filter.test.js]: (Resolved by b8019ac.)

**Pull request merged**